doc/classes/NodePath.xml

```diff
@@ -4,25 +4,33 @@
+		A node path is represented as a [String] composed of slash-separated ([code]/[/code]) node names and colon-separated ([code]:[/code]) property names (also called "subnames"). Similar to a filesystem path, [code]".."[/code] and [code]"."[/code] are special node names. They refer to the parent node and the current node, respectively.
+		The following examples are paths relative to the current node:
 		[codeblock]
-		# No leading slash means it is relative to the current node.
-		^"A" # Immediate child A
-		^"A/B" # A's child B
-		^"." # The current node.
-		^".." # The parent node.
-		^"../C" # A sibling node C.
-		^"../.." # The grandparent node.
-		# A leading slash means it is absolute from the SceneTree.
-		^"/root" # Equivalent to get_tree().get_root().
-		^"/root/Main" # If your main scene's root node were named "Main".
-		^"/root/MyAutoload" # If you have an autoloaded node or scene.
+		^"A"     # Points to the direct child A.
+		^"A/B"   # Points to A's child B.
+		^"."     # Points to the current node.
+		^".."    # Points to the parent node.
+		^"../C"  # Points to the sibling node C.
+		^"../.." # Points to the grandparent node.
 		[/codeblock]
-		See also [StringName], which is a similar concept for general-purpose string interning.
-		[b]Note:[/b] In the editor, [NodePath] properties are automatically updated when moving, renaming or deleting a node in the scene tree, but they are never updated at runtime.
+		A leading slash means the path is absolute, and begins from the [SceneTree]:
+		[codeblock]
+		^"/root"            # Points to the SceneTree's root Window.
+		^"/root/Title"      # May point to the main scene's root node named "Title".
+		^"/root/Global"     # May point to an autoloaded node or scene named "Global".
+		[/codeblock]
+		Despite their name, node paths may also point to a property:
+		[codeblock]
+		^"position"            # Points to this object's position.
+		^"position:x"          # Points to this object's position in the x axis.
+		^"Camera3D:rotation:y" # Points to the child Camera3D and its y rotation.
+		^"/root:size:x"        # Points to the root Window and its width.
+		[/codeblock]
+		Node paths cannot check whether they are valid and may point to nodes or properties that do not exist. Their meaning depends entirely on the context in which they're used.
+		You usually do not have to worry about the [NodePath] type, as strings are automatically converted to the type when necessary. There are still occasions when defining node paths is useful. For example, exported [NodePath] properties allow you to easily select any node within the currently edited scene. They are also automatically updated when moving, renaming or deleting nodes in the scene tree editor. See also [annotation @GDScript.@export_node_path].
```

Comment:
```suggestion
		You usually do not have to worry about the [NodePath] type, as strings are automatically converted to the type when necessary. There are still times when defining node paths is useful. For example, exported [NodePath] properties allow you to easily select any node within the currently edited scene. They are also automatically updated when moving, renaming or deleting nodes in the scene tree editor. See also [annotation @GDScript.@export_node_path].
```
Prefer more common terms in the documentation when appropriate

Reply:
That is indeed the goal. Thank you